### ISSUE TITLE
[P2][Sourcing][Inline-help][Accessibility][13386] : When Inline help receives focus screenreader is not reading role as tooltip

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#13440]
### Describe your proposal

Reporting the issue again as https://github.com/SAP/fundamental-ngx/issues/13386 is closed without any fix.
Its clealy mention on w3c standards as per accessibility that container of tooltip should have role as tooltip.
 https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/ "The element that serves as the tooltip container has role [tooltip](https://w3c.github.io/aria/#tooltip)."

### Which versions of Angular and Fundamental Library for Angular are affected?

0.43.57

### Steps to Reproduce (for bugs)

Please check https://github.com/SAP/fundamental-ngx/issues/13386

### StackBlitz example(s)

https://ng-15-downport--fundamental-ngx.netlify.app/#/core/inline-help 

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by droshev** [comment]
@I543348 As far as I see there is role set to tooltip in ng-15-downport

<img width="1533" height="866" alt="Image" src="https://github.com/user-attachments/assets/6f83330b-1740-4cd3-aff0-271457dc8708" />

**Comment by I543348** [comment]
Hi @droshev ,
Thanks for the details.
The popover is actually the tooltip however, if you check w3 standards its not the tooltip but the "tooltip container" needs to have this role  https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/ 
The element that serves as the tooltip container has role [tooltip](https://w3c.github.io/aria/#tooltip).

**Comment by droshev** [comment]
@I543348 the spec is not finalized: `NOTE: This design pattern is work in progress; it does not yet have task force consensus.` SAP accessibility is also discussing this topic and it is work in progress. Once this is finalized, we can plan to implement it.

<img width="917" height="182" alt="Image" src="https://github.com/user-attachments/assets/4ff6df46-19c1-40af-86bf-ac33ff77b6c3" />

**Comment by droshev** [comment]
Closing the issue as we got input from the accessibility experts that there is no outstanding issue.

**Comment by I543348** [comment]
@droshev  pls reopen this , Stefan comment clearly says , tooltip role is needed

**Comment by droshev** [comment]
@I543348 If you see the [example](https://ng-15-downport--fundamental-ngx.netlify.app/#/core/inline-help#trigger-event) we set that role.

<img width="2934" height="596" alt="Image" src="https://github.com/user-attachments/assets/c85c468f-31cd-4ed0-9921-de3c3b2b5108" />

**Comment by I543348** [comment]
> example

Hi @droshev , but its not working for this example https://ng-15-downport--fundamental-ngx.netlify.app/#/core/inline-help#basic when we dont have  [triggers]="['click']"

**Comment by droshev** [comment]
@I543348 I asked a question inside our common chat.

**Comment by I543348** [comment]
> [@I543348](https://github.com/I543348) I asked a question inside our common chat.

Thanks @droshev  , i have replied over chat internally

**Comment by droshev** [comment]
@I543348 
I have listened to our colleague's comments from the video. He iterated few points:
 
- REMOVE **role=“tooltip”** from the icon (this is something that YOU need to remove from your application)
- ​ADD **role=“img”** to the icon (this is something that YOU need to add)
​- ADD **arai-label** to the same element (this is something that YOU need to add) - see the first screenshot if you don’t know how to do it.
- REMOVE **aria-labeledby** from icon (this is something that YOU need to remove)
- The focus should stay on the icon and NOT to the tooltip itself - this is how it is right now
- The tooltip container(the one that show the tooltip) should have the **role=“tooltip”** - this is the case already
- It is up to the screen reader application whether it should pronounce it or not. - the control of the reader, not us.

<img width="1593" height="163" alt="Image" src="https://github.com/user-attachments/assets/6eb20124-822f-4ae8-a8a8-ce9fe8bd1b31" />
<img width="1609" height="286" alt="Image" src="https://github.com/user-attachments/assets/bde8411f-151d-4ac9-aea2-96cb93645459" />
<img width="1327" height="571" alt="Image" src="https://github.com/user-attachments/assets/e04db499-b183-41ed-af22-ae28b148b9c3" />

With that I consider the support from our side done.
